apps/functions-subscription/src/config.ts

```diff
@@ -81,58 +85,109 @@ const EnvsCodec = t.strict({
   SERVICE_BUS_LOCATION: NonEmptyString,
 });
 
+const DevelopmentCodec = t.strict({
+  NODE_ENV: t.literal('development'),
+  SubscriptionHistoryCosmosConnection: NonEmptyString,
+  SubscriptionRequestEventHubConnection: NonEmptyString,
+  ActivationConsumerCosmosDBConnection: NonEmptyString,
+  TrialsCosmosConnection: NonEmptyString,
+});
```

Comment:
**note**: Doing this we are forcing to set these env variables when running the app function with `NODE_ENV` set to `development` (e.g. running locally)